config/webhook/service.yaml

```diff
@@ -0,0 +1,13 @@
+
+apiVersion: v1
+kind: Service
+metadata:
+  name: webhook-service
+  namespace: system
+spec:
+  ports:
+    - port: 443
+      protocol: TCP
+      targetPort: 9443
+  selector:
+    control-plane: controller-manager
```

Comment:
don't we have it already in https://github.com/metal3-io/baremetal-operator/blob/master/config/webhook/service_patch.yaml  ?

Reply:
Indeed. Removing it.